uxpin-merge/template-merge-typescript-ds
# Add code format tools

## Goal

Currently we do not have code formatting rules in this repo. We want to have consistency between all company repos to reduce DX friction for developers and to display a well kept repository for customers.

Added the following tools:
[prettier](https://prettier.io/)
https://eslint.org/

## How to check

- Ensure dependencies are up-to-date: `yarn install`
- Ensure your VSCode has the "recommended" extensions: ESLint and Prettier
- Edit any TS file, removing an ending `;`, changing the spacing, adding tabs...
- Save
- The `;` should be back automatically, the file should be in its initial state, with no change
